### PR TITLE
add filtering option for Net devices

### PR DESCRIPTION
**Pull request by vindex10**
Add option to specify which network devices to show. The option is relevant for instance when docker network interfaces are not of interest:

```
× sit0@NONE
√ wlp3s42
√ enp0s20f41
× br-uh14b3h3v
× docker0
× ethekjbrh14@if7
```
Example:
```
DynNetwork ["--", "--devices", "wlp3s42,enp0s20f41"] 10
```

### CODE REVIEW

File: src/Xmobar/Plugins/Monitors/Common/Output.hs [hunk 1]
```diff
@@ -17,6 +17,8 @@
 
 module Xmobar.Plugins.Monitors.Common.Output ( IconPattern
                                              , parseIconPattern
+                                             , DevList
+                                             , parseDevList
```
Comment (jaor) [inline src/Xmobar/Plugins/Monitors/Common/Output.hs]: this is a minor thing, but i think that, given that parseDevList is used only in Net.hs, i would define it there (as well as DevList) and not even export it.

**Comment by jaor** [comment]
another minor thing, but perhaps we could call the switch `--devices`, which is shorter and easier to remember?

**Comment by jaor** [comment]
oh, and would you add a mention of the new option to the corresponding section in the readme? thanks a lot!

**Comment by jaor** [comment]
squashed and merged, thanks a lot!

**Comment by vindex10** [comment]
my pleasure!)

**Pull request merged**